.github/workflows/node.js.yml

```diff
@@ -1,13 +1,8 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
-
-name: Node.js CI
+name: CI
```

Comment:
Let's keep the name the same just for continuity and to keep the scope of this PR solely to formatting changes!